lang/de/local_lsf_unification.php

```diff
@@ -166,4 +166,13 @@
 $string['eventcourse_imported'] = 'Kurs importiert';
 $string['eventmatchingtable_updated'] = 'Matchingtable ge&auml;ndert';
 $string['eventcourse_duplicated'] = 'Kursinhalte dupliziert';
-
+// Privacy API
+$string['privacy:metadata:local_lsf_unification:veranstid'] = 'ID der Veranstaltung im HisLSF System';
+$string['privacy:metadata:local_lsf_unification:mdlid'] = 'ID der Veranstaltung im Moodle System';
+$string['privacy:metadata:local_lsf_unification:timestamp'] = 'Zeitpunkt yu dem der Kurs importiert wurde';
+$string['privacy:metadata:local_lsf_unification:requeststate'] = 'Status der Kursanfrage';
+$string['privacy:metadata:local_lsf_unification:requesterid'] = 'ID der Person die den Kurs angefragt hat';
+$string['privacy:metadata:local_lsf_unification:acceptorid'] = 'ID der Person die den Kurs akzeptiert hat';
+
+$string['privacy:local_lsf_unification:requester'] = 'Die Anfrage f&uuml;r einen Kurs wurde gespeichert.';
```

Comment:
This string, and the following, and the corresponding ones in the english version are not used.

Reply:
Ohhhh, they are, but indirectly. Sorry. Hm, that could be clarified in a comment.  Acceptor could be explained as 
"Eine Kursanfrage wurde akzeptiert" instead.

Reply:
The course can also be declined therefore I chose for the cumbersome formulation.